static/js/components/HomePage.jsx

```diff
@@ -1,29 +1,108 @@
 import React from "react";
-import { useSelector } from "react-redux";
+import { useSelector, useDispatch } from "react-redux";
 
+import { Responsive, WidthProvider } from "react-grid-layout";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import "reactgridlayoutcss/styles.css";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import "reactresizablecss/styles.css";
+
+import * as profileActions from "../ducks/profile";
 import SourceList from "./SourceList";
 import GroupList from "./GroupList";
 import NewsFeed from "./NewsFeed";
 import TopSources from "./TopSources";
 import styles from "./HomePage.css";
 
+const ResponsiveGridLayout = WidthProvider(Responsive);
+
+const xlgLayout = [
+  { i: "sourceList", x: 0, y: 0, w: 9, h: 6, minW: 9, isResizable: false },
```

Comment:
This is great, but eventually we would probably want to move this over to a config of some sort. What say you, @acrellin?